services/admin/backend/api/handler/project.go

```diff
@@ -377,6 +377,110 @@ func deleteProject(service project.UseCase) http.Handler {
 	})
 }
 
+func migrateProject(service project.UseCase) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		errorMessage := "Error creating project"
+		var input struct {
+			ShortCode   string `json:"shortCode"`
+			ShortName   string `json:"shortName"`
+			LongName    string `json:"longName"`
+			Description string `json:"description"`
```

Comment:
Since this struct is used over few functions here it should be defined once and reused.